convert2rhel/breadcrumbs.py

```diff
@@ -71,6 +71,8 @@ def __init__(self):
         self._pkg_object = None
         # Flag to inform the user about DISABLE_TELEMETRY. If not informed, we shouldn't save rhsm_facts.
         self._inform_telemetry = False
+        # Record what type of convert2rhel run we are performing.  Valid options right now are convert or analyze.
+        self.action = "convert"
```

Comment:
Commenting [as requested](https://github.com/oamg/convert2rhel/pull/806#issuecomment-1509089598) :)

> @r0x0d, have you checked if leapp has action as well. Or how do they specify in the breadcrumbs that a pre-upgrade analysis is being performed. We don't want to diverge from them to have the same migration breadcrumbs output.

> Without knowing how leapp does that, it seems to me the breadcrumbs data before this change already contained the information whether convert2rhel has been invoked in an analysis mode or not - through the env field. Later when we switch to using the sub-command, it will be captured under executed.

Reply:
@abadger probably something to discuss in our 1:1

Reply:
I don't think it's good to rely on env or executed for this.  env is not going to be there once we switch to a subcommand and wouldn't work if we had two modes of operation already written.  executed would require parsing the field to look for the command.

It looks like leapp uses the field `activity` for this.

Reply:
(I have some changes to implement this.  I'm just going to commit r0x0d's easy suggestions first)

Reply:
Okay this has been implemented as two commits:
* https://github.com/oamg/convert2rhel/pull/761/commits/b72db596177575aa3eccbe75dd603e2537eeb0a9  -- Move determination of the activity into toolopts.  

* https://github.com/oamg/convert2rhel/pull/761/commits/1aecabf526da434fd47a91a1ea5c4184db789a68 -- Remove the action field and enhance the activity field to take its place.